tests/integration/operator/switch_cr_test.go

```diff
@@ -77,6 +79,9 @@ func TestController(t *testing.T) {
 				t.Fatal(err)
 			}
 			cleanupInClusterCRs(t, cs)
+			t.Cleanup(func() {
+				cleanupIstioResources(t, cs, istioCtl)
+			})
```

Comment:
Is it necessary to manually cleanup the istio resources after all the istio operator CRs are deleted, the in-cluster operator will cleanup the istio resources. right?

Reply:
@morvencao makes sense, we should probably check whether the istio resources are cleaned up as part of test after deleting the CR. But anyways there are still some other resources that require manually clean up, for example the secrets/configmaps created dynamically by istiod.